spec/integration/api_resource_comments_controller_spec.rb

```diff
@@ -64,4 +64,71 @@
       end
     end
   end
+
+  describe 'Resources Comments API' do
+    path '/api/resources/{resource_id}/resource_comments' do
+      post 'Creates a new comment' do
+        tags 'Resource comment'
+        consumes 'application/json'
+        parameter name: :resource_id, in: :path, type: :string
+        parameter name: :json_body, in: :body, schema: {
+          type: :object,
+          properties: {
+            content: { type: :string }
+          },
+          required: ['content']
+        }
+        produces 'application/json'
+        operationId 'createResourceComment'
+
+        response '201', 'Success' do
+          schema type: :object,
+                 properties: {
+                   id: { type: :integer },
+                   content: { type: :string },
+                   resource_id: { type: :integer }
+                 }
+          let(:resource_id) { create(:resource).id }
+          let(:json_body) { { 'content': 'test' } }
+          run_test!
+        end
```

Comment:
I think we should consider the test case that if the response is 201, the comment is effectively created, that is, the table `resource_comments` count increase by 1.

Reply:
I've included a validation checking the last comment content and the response content. Thanks!